equinix/resource_fabric_connection.go

```diff
@@ -394,14 +394,14 @@ func resourceFabricConnectionDelete(ctx context.Context, d *schema.ResourceData,
 		return diag.FromErr(equinix_errors.FormatFabricError(err))
 	}
 
-	err = waitUntilConnectionDeprovisioned(d.Id(), meta, ctx)
+	err = WaitUntilConnectionDeprovisioned(d.Id(), meta, ctx)
```

Comment:
Same as above, this is made public in-place in order to separate the customer-facing code from the acceptance tests with minimal changes to the customer-facing code.